vercel/turbo
# chore: use vendored vt100 for tui-term

### Description

https://github.com/a-kenji/tui-term/pull/152 allowed for the widget to be used with any library that implements the `Screen` trait. This PR switches us to use our vendored vt100 crate as the backend for the terminal widget.

If desired should be easy to review each commit individually.

### Testing Instructions

`cargo run -p turborepo-ui --example pane` and verify that colors looks nice


Closes TURBO-2623
